tools/c7n_azure/c7n_azure/filters.py

```diff
@@ -564,7 +564,8 @@ class FirewallRulesFilter(Filter, metaclass=ABCMeta):
             'include': {'type': 'array', 'items': {'type': 'string'}},
             'any': {'type': 'array', 'items': {'type': 'string'}},
             'only': {'type': 'array', 'items': {'type': 'string'}},
-            'equal': {'type': 'array', 'items': {'type': 'string'}}
+            'equal': {'type': 'array', 'items': {'type': 'string'}},
+            'include_azure_services_rule': {'type': 'boolean'}
```

Comment:
afaics, this provider generally uses hyphenated names instead of underscore for filter parameters. 

re naming `include-azure-services` ?

also worth documenting.

Reply:
Updated to kebab-case and added a doc example